package.json

```diff
@@ -683,7 +683,7 @@
         },
         {
           "command": "confluent.resources.schema-registry.select",
-          "when": "view == confluent-schemas",
+          "when": "view == confluent-schemas && (confluent.ccloudConnectionAvailable || confluent.localSchemaRegistryAvailable || confluent.directSchemaRegistryAvailable)",
```

Comment:
I'd like to update `confluent.ccloudConnectionAvailable` to `confluent.ccloudSchemaRegistryAvailable` (and same for the Kafka variant) but that can be in another PR since it may be slightly more involved to implement.

That same follow-on PR could also turn these checks into a single contextValue configured at activation:
```
confluent.ccloudSchemaRegistryAvailable || confluent.localSchemaRegistryAvailable || confluent.directSchemaRegistryAvailable
```
Similar to how we set other custom contextValues: https://github.com/confluentinc/vscode/blob/40816c4cecf0b107545ec16f91d3e61a4a7275e0/src/extension.ts#L246-L252